orchestration/pom.xml

```diff
@@ -9,36 +9,83 @@
   <artifactId>orchestration</artifactId>
   <name>Orchestration client</name>
   <description>Generating and testing the Orchestration client.</description>
+
+  <url>https://github.com/SAP/ai-sdk-java?tab=readme-ov-file#documentation</url>
   <organization>
     <name>SAP SE</name>
     <url>https://www.sap.com</url>
   </organization>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <name>SAP</name>
+      <email>cloudsdk@sap.com</email>
+      <organization>SAP SE</organization>
+      <organizationUrl>https://www.sap.com</organizationUrl>
+    </developer>
+  </developers>
 
   <dependencies>
+    <!-- scope "compile" -->
     <dependency>
       <groupId>com.sap.cloud.sdk.datamodel</groupId>
       <artifactId>openapi-core</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.sap.ai.sdk</groupId>
-      <artifactId>core</artifactId>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+      <artifactId>cloudplatform-connectivity</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
     </dependency>
     <!-- scope "test" -->
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter</artifactId>
+      <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.wiremock</groupId>
       <artifactId>wiremock</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <scope>test</scope>
```

Comment:
I would expect this to be `compile` or `runtime`

Reply:
HttpClient5 is compile, HttpClient4 is used tests like for `org.apache.http.HttpStatus.SC_BAD_REQUEST`

Reply:
With the declaration above you are degrading the implicit transitive scope "runtime" to explicit direct scope "test".

As a result the other code that may rely on HttpClient 4 to be present (e.g. Destination Service lookup, Token exchanges) will no longer work productively, since classes are missing.

<details><summary>Dependency tree from orchestration module</summary>

![image](https://github.com/user-attachments/assets/870079bc-4ada-4146-b780-f71133d0dd0d)


</details>

Reply:
I think that is fine, relevant is not the scope within the orchestrationm odule iteself, relevant is the scope that is in effect when using the orchestration module. Confusingly, those are two different things 😄 

However, we need the OAuth dependency to come in by default, e.g. by relying on `sdk-core` with compile scope in the orchestration module or any of its dependencies (thus probably in the core module after we start using that in compile scope).

Reply:
> I think that is fine, relevant is not the scope within the orchestrationm odule iteself, relevant is the scope that is in effect when using the orchestration module. Confusingly, those are two different things

You're right, (in cases like this) the dependency remains `compile` when using the module. I confirmed with a sample project.

<details><summary>For next time I will use the "Effective POM" view</summary>

![image](https://github.com/user-attachments/assets/94f715bd-4d70-49e2-9c13-41693af8fc8e)


<details/>